data/json/obsoletion/obsolete-0.G-items.json

```diff
@@ -262,5 +262,51 @@
         "encumbrance": 20
       }
     ]
+  },
+  {
+    "id": "bio_power_armor_interface",
```

Comment:
Moving stuff around in files is not going to make a difference, you need  to make 2 MIGRATION entries like described in https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/JSON_INFO.md#obsoletion-and-migration to some inert item type id (say burnt bionic or whatever else), or alternatively another bionic id of same rarity and theme

Also these files are for the 0.G obsoletions that are left over still, new items shouldn't be put here